modules/phase_field/src/utils/PolycrystalICTools.C

```diff
@@ -177,7 +177,7 @@ PolycrystalICTools::buildElementalGrainAdjacencyGraph(const std::map<dof_id_type
   {
     std::set<dof_id_type> orig_halo_ids(halo_ids[i]);
 
-    for (unsigned int halo_level = 0; halo_level < 2; ++halo_level)
+    for (unsigned int halo_level = 0; halo_level < 4; ++halo_level)
```

Comment:
Yeah... I was debating on making this a parameter. That's why I haven't bothered to make it a constant yet.